cfme/v2v/migrations.py

```diff
@@ -262,13 +267,32 @@ def after_fill(self, was_change):
             self.result.close_btn.click()
 
 
+# Widget for migration selection dropdown
+class MigrationDropdown(Dropdown):
+    """Represents the migration plan dropdown of v2v.
+
+    Args:
+        text: Text of the button, can be inner text or the title attribute.
+    """
+    ROOT = './/div[contains(@class, "dropdown") and .//button[contains(@id, "dropdown-filter")]]'
+    BUTTON_LOCATOR = './/button[contains(@id, "dropdown-filter")]'
+    ITEMS_LOCATOR = './/ul[contains(@aria-labelledby,"dropdown-filter")]/li/a'
+    ITEM_LOCATOR = './/ul[contains(@aria-labelledby,"dropdown-filter")]/li/a[normalize-space(.)={}]'
+
+    def __init__(self, parent, text, logger=None):
```

Comment:
**REQUIRED:** This is not needed as there is no difference between this and the inherited class